code/modules/spells/spell_types/mime.dm

```diff
@@ -174,45 +174,36 @@
 		invocation_type ="none"
 	..()
 
-/obj/effect/proc_holder/spell/aimed/finger_guns
+/obj/effect/proc_holder/spell/targeted/mime/finger_guns
 	name = "Finger Guns"
 	desc = "Shoot a mimed bullet from your fingers that stuns and does some damage."
 	school = "mime"
 	panel = "Mime"
 	charge_max = 300
 	clothes_req = FALSE
 	antimagic_allowed = TRUE
+	include_user = TRUE
 	invocation_type = "emote"
 	invocation_emote_self = "<span class='dangers'>You fire your finger gun!</span>"
-	range = 20
-	projectile_type = /obj/item/projectile/bullet/mime
-	projectile_amount = 3
 	sound = null
-	active_msg = "You draw your fingers!"
-	deactive_msg = "You put your fingers at ease. Another time."
-	active = FALSE
 
 	action_icon = 'icons/mob/actions/actions_mime.dmi'
 	action_icon_state = "finger_guns0"
 	action_background_icon_state = "bg_mime"
-	base_icon_state = "finger_guns"
-
 
-/obj/effect/proc_holder/spell/aimed/finger_guns/Click()
-	var/mob/living/carbon/human/owner = usr
-	if(owner.incapacitated())
-		to_chat(owner, "<span class='warning'>You can't properly point your fingers while incapacitated.</span>")
+/obj/effect/proc_holder/spell/targeted/mime/finger_guns/Click()
+	if(!usr)
 		return
-	if(usr?.mind)
-		if(!usr.mind.miming)
```

Comment:
Should probably keep the check if their vow of silence is broken. Either that or this is an undocumented buff to them.

Reply:
View commit sha bcc30a0